awx/main/isolated/manager.py

```diff
@@ -268,13 +268,6 @@ def check(self, interval=None):
         # in the final sync
         self.consume_events()
 
-        # emit an EOF event
```

Comment:
Another bit of code duplication and an opportunity to converge the standard and isolated event handling paths.